ckanext/unaids/tests/test_dataset_transfer.py

```diff
@@ -97,31 +97,49 @@ def test_get_org_admins_with_no_users_in_org(self, app):
             )
 
     @mock.patch('ckan.lib.mailer.mail_user')
-    def test_send_dataset_transfer_emails(self, mocked_mail_user, app):
-        user_1, user_2 = [
-            factories.User()
-            for x in range(2)
-        ]
-        org_1, org_2 = [
-            factories.Organization(user=user)
-            for user in [user_1, user_2]
-        ]
+    @mock.patch('ckan.lib.mailer._mail_recipient')
+    def test_send_dataset_transfer_emails(self, mocked_mail_recipient, mocked_mail_user, app):
+        user_1 = factories.User(email='user_1@example.com')
+        user_2 = factories.User(email='user_2@example.com')
```

Comment:
why this approach instaed of the older one:
```
        user_1, user_2 = [
            factories.User()
            for x in range(2)
        ]
```
You're asiging spcific e-mails to those test users but they're not used afterwards. If it's the case it'd be better to leave them with default values.

Reply:
I couldn't get tests passing without doing this, otherwise I would have left it. I know it doesn't make too much sense but my guess was that you need to specify an email address as the factory wasn't doing a good enough job to avoid errors.